webui/react/src/hooks/useModal/useModalCustomizeColumns.tsx

```diff
@@ -100,15 +101,40 @@ const useModalCustomizeColumns = ({
     );
   }, [ renderColumnName ]);
 
+  const switchRowOrder = useCallback((itemToMoveIndex:number, newItemIndex:number) => {
+    if(itemToMoveIndex !== newItemIndex){
+      const updatedVisibleColumns = [ ...visibleColumns ];
+      const columnToMove = updatedVisibleColumns[newItemIndex];
+      updatedVisibleColumns.splice(newItemIndex, 1);
+      updatedVisibleColumns.splice(itemToMoveIndex, 0, columnToMove);
+      setVisibleColumns(updatedVisibleColumns);
+    }
+    return;
+  }, [ visibleColumns ]);
+
+  const renderDraggableRow = useCallback((index, row, style, handleClick, handleDrop) => {
+    return (
+      <DraggableListItem index={index} style={style} onClick={handleClick} onDrop={handleDrop}>
+        {renderColumnName(row)}
+      </DraggableListItem>
+    );
+  }, [ renderColumnName ]);
+
   const renderHiddenRow = useCallback(({ index, style }) => {
     const row = filteredHiddenColumns[index];
     return renderRow(row, style, () => makeVisible(row));
   }, [ filteredHiddenColumns, makeVisible, renderRow ]);
 
   const renderVisibleRow = useCallback(({ index, style }) => {
     const row = filteredVisibleColumns[index];
```

Comment:
`nit`/ `question` do you think it would make the code easier to read if all instances of `row`/`Row` were changed to `column`/`Column`. I get that the actual ui elements are rows of a list, but just wanted to throw it out there.

Reply:
I initially found this confusing because the terms `row` and `column` usually refer to the same matrix or table. But here we are referencing a list `row` from the list of `columnNames`. I updated several variables to use `columnName` instead of just `column` which makes things clearer to me.